### ISSUE TITLE
"safety borders"

### ISSUE THREAD

**Issue by jlsjonas** [eczarny/spectacle#820]
Allow configuring a "safety border", when using spectacle to resize windows; it should stay clear of this area.

Example use-case: Play music sits nicely in the background, taking up a small amount of my screen real-estate on the bottom. Sadly, this gets covered by moving a window with spectacle.

Most use-cases will likely be covered by picking a side and amount of px/%.
(abusing the dock could technically become a workaround, but the dock would be in the way then)

**Comment by aamave** [comment]
I would agree. When it occupies 100% height, it becomes a challenge to reach the bottom of a window without triggering a hidden dock to pop up. Perhaps that space at the bottom could be a preference? (that's where I went to look for it)
